lib/routes/content.js

```diff
@@ -215,7 +215,7 @@ const routes = application => {
             return types.form(merged, errors, config).then(form => {
               res.render('content/add', {
                 form,
-                action: `/${config.content.base}/${req.params.type}/${config.content.actions.save}`,
+                action: `/${config.content.base}/${req.params.type.toLowerCase()}/${config.content.actions.save}`,
```

Comment:
@Snugug cannot test this because the routes are not yet pulled out of the `content` section. [this comment on the refactor story](https://github.com/punchcard-cms/punchcard/issues/218#issuecomment-249609951) captures the need to add a test that the `action` attribute is what we expect it to be.

ie...cannot add a test now, unless we make it an E2E test in `server.js`. Which is dumb. I hate that file. I haet
![ihaetusohard](https://cloud.githubusercontent.com/assets/216931/18841402/971d44fc-83df-11e6-89ca-14bad024d586.jpg)
 that file so hard.